crates/bitwarden-vault/src/cipher/ssh_key.rs

```diff
@@ -41,9 +41,21 @@ impl KeyEncryptable<SymmetricCryptoKey, SshKey> for SshKeyView {
 impl KeyDecryptable<SymmetricCryptoKey, SshKeyView> for SshKey {
     fn decrypt_with_key(&self, key: &SymmetricCryptoKey) -> Result<SshKeyView, CryptoError> {
         Ok(SshKeyView {
-            private_key: self.private_key.decrypt_with_key(key).ok().flatten(),
-            public_key: self.public_key.decrypt_with_key(key).ok().flatten(),
-            fingerprint: self.fingerprint.decrypt_with_key(key).ok().flatten(),
+            private_key: Some(self.private_key.clone())
+                .decrypt_with_key(key)
+                .ok()
+                .flatten()
+                .ok_or(CryptoError::MissingField("private_key"))?,
+            public_key: Some(self.public_key.clone())
+                .decrypt_with_key(key)
+                .ok()
+                .flatten()
+                .ok_or(CryptoError::MissingField("public_key"))?,
+            fingerprint: Some(self.fingerprint.clone())
+                .decrypt_with_key(key)
+                .ok()
+                .flatten()
+                .ok_or(CryptoError::MissingField("fingerprint"))?,
```

Comment:
This seems a bit convoluted, we should be able to just call decrypt on the items, unless I'm missing something
```suggestion
            private_key: self.private_key.decrypt_with_key(key)?,
            public_key: self.public_key.decrypt_with_key(key)?,
            fingerprint: self.fingerprint.decrypt_with_key(key)?,
```